src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
```

Comment:
do not use padding and margin reset on the `*` selector. u can list all the tags u need.
use `*` for `box-sizing: border-box` only
For example:
```
body,
p,
h1 {
  margin: 0;
}
```
```suggestion
```